CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.2] - 2021-03-05
+
+### Changed
```

Comment:
"Added" seems more appropriate here:

```suggestion
### Added
```

The existing methods weren't changed in any way.

Reply:
Good point!